tv_apps/tv-deck/js/tv_deck.js

```diff
@@ -62,15 +63,17 @@
 
     this.simpleKeyNavigation = new SimpleKeyNavigation();
 
-    this.pinCard = new PinCard(this.selfApp);
-    this.pinCard.on('update-pin-button', this.updatePinButton.bind(this));
+    this.contextmenu = new Contextmenu([{
```

Comment:
I cannot understand the argument of Contextmenu. We should have two buttons: pin/unpin and delete button. Why does argument have two pinButton here? And where is the delete button?